vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/SpaceManagerTest.java

```diff
@@ -116,23 +125,27 @@ public void test_storeSpace_success() throws FileStorageException {
         Mockito.verify(spaceDisplayRepo).save((SpaceDisplay)spaceDisplay);
     }
 
-
-
-    @Test(expected = SpaceDoesNotExistException.class)
-    public void test_deleteSpaceById_whenIdIsNull() throws SpaceDoesNotExistException {
-        Mockito.doThrow(IllegalArgumentException.class)
-        .when(spaceRepo).deleteById(null);
+    @Test
+    public void test_deleteSpaceById_whenIdIsNull() throws SpaceDoesNotExistException{
         managerToTest.deleteSpaceById(null);
+        Mockito.verify(spaceDisplayRepo).deleteBySpaceId(null);
+        Mockito.verify(spaceRepo).deleteById(null);
```

Comment:
why is the manager called delete function with null?

Reply:
Should I remove the null test case then? We do not have a null checker in the deleteSpaceById method in spaceManager. Or should I add null checker in the method, then test that the spaceRepo.deleteById(null) is not called ?